### PR TITLE
Automate release process

### PR DESCRIPTION
**Pull request by ndeloof**
Fully automate the release based on git tag.
Only need to update compose/__init__.py for released version before tagging
(maybe we could use https://pypi.org/project/bumpversion/)

see test release https://github.com/docker/compose/releases/tag/untagged-86c6d2315f95ab28e779

### CODE REVIEW

File: tests/fixtures/simple-failing-dockerfile/Dockerfile [hunk 1]
```diff
@@ -2,6 +2,5 @@ FROM busybox:1.31.0-uclibc
 LABEL com.docker.compose.test_image=true
 LABEL com.docker.compose.test_failing_image=true
 # With the following label the container wil be cleaned up automatically
-# Must be kept in sync with LABEL_PROJECT from compose/const.py
```
Comment (ulyssessouza) [inline tests/fixtures/simple-failing-dockerfile/Dockerfile]: I didn't understand this removal

File: script/release/README.md [hunk 2]
```diff
@@ -1,201 +1,8 @@
 # Release HOWTO
 
-This file describes the process of making a public release of `docker-compose`.
-Please read it carefully before proceeding!
+The release process is fully automated by `Release.Jenkinsfile`.
 
-## Prerequisites
+## usage
```
Comment (ulyssessouza) [inline script/release/README.md]:
```suggestion
## Usage
```

File: script/release/README.md [hunk 3]
```diff
@@ -1,201 +1,8 @@
...
-      update it to `1.5.0dev`
-    - Update compose_version in [github.com/docker/docker.github.io/blob/master/_config.yml](https://github.com/docker/docker.github.io/blob/master/_config.yml) and [github.com/docker/docker.github.io/blob/master/_config_authoring.yml](https://github.com/docker/docker.github.io/blob/master/_config_authoring.yml)
-    - Update the release note in [github.com/docker/docker.github.io](https://github.com/docker/docker.github.io/blob/master/release-notes/docker-compose.md)
-
-## Advanced options
-
-You can consult the full list of options for the release tool by executing
-`./script/release/release.sh --help`.
+1. create a tag for release
+1. done
```
Comment (ulyssessouza) [inline script/release/README.md]: Not needed

File: script/release/README.md [hunk 4]
```diff
@@ -1,201 +1,8 @@
...
-      number with `dev` appended. For example, if you just released `1.4.0`,
-      update it to `1.5.0dev`
-    - Update compose_version in [github.com/docker/docker.github.io/blob/master/_config.yml](https://github.com/docker/docker.github.io/blob/master/_config.yml) and [github.com/docker/docker.github.io/blob/master/_config_authoring.yml](https://github.com/docker/docker.github.io/blob/master/_config_authoring.yml)
-    - Update the release note in [github.com/docker/docker.github.io](https://github.com/docker/docker.github.io/blob/master/release-notes/docker-compose.md)
-
-## Advanced options
-
-You can consult the full list of options for the release tool by executing
-`./script/release/release.sh --help`.
+1. create a tag for release
```
Comment (ulyssessouza) [inline script/release/README.md]: Could you please, describe the tagging format?

File: tests/fixtures/simple-failing-dockerfile/Dockerfile (hunk 1 above)
Comment (ndeloof) [reply tests/fixtures/simple-failing-dockerfile/Dockerfile]: PyCharm went confused during refactoring it seems :P

File: Jenkinsfile [hunk 5]
```diff
@@ -1,8 +1,8 @@
 #!groovy
 
-def dockerVersions = ['19.03.5', '18.09.9']
+def dockerVersions = ['19.03.5']
```
Comment (ulyssessouza) [inline Jenkinsfile]: Why removing the second docker version?

Comment (ndeloof) [reply Jenkinsfile]: 19.09 is EOL (according to @thaJeztah)

**Pull request merged**